### ISSUE TITLE
Expose Open Volumes feature should be configurable for each volume

### ISSUE THREAD

**Issue by Murilogs1910** [hardcore-sushi/DroidFS#240]
After enabling the feature `Allow saving password hash using fingerprint`, that is not automatically enabled for every volume. Instead, the user can choose which volumes' password hashes are stored that way.

The same should be done for the `Expose open volumes` feature. The user should be able to choose for each volume whether they are exposed or not, and whether they are read-only or if the apps also have write access to it.

**Comment by hardcore-sushi** [comment]
Yes, that would be a good improvement. How do you think the user should be able to configure this?
